### ISSUE TITLE
Reporter: Report generation fails via CLI when db file and template have not finished loading (using GUI)

### ISSUE THREAD

**Issue by pkievit86** [damies13/rfswarm#132]
**Describe the bug**
Reporter - Report generation fails via CLI when db file and template have not finished loading (using GUI)

**To Reproduce**

rfswarm_reporter -i 'RFSwarmReporter.ini' -d '20230221_100714_qconnect-load-test.db' --debug 1 -t 'default.template' --html --xlsx --docx

**Expected behavior**
Reporter needs to take into account any loading delays and generate the report.

**Log**
Default ini with:

[GUI]
win_width = 800
win_height = 390

patrick@MacBook-Pro-van-Patrick ~ % rfswarm_reporter -i '/Users/patrick/Library/Application Support/Copado/QConnect/execution/performance-testing/files/configuration/RFSwarmReporter.ini' -d '/Users/patrick/Library/Application Support/Copado/QConnect/execution/performance-testing/20230221_100714_qconnect-load-test/20230221_100714_qconnect-load-test.db' --debug 1 -t '/Users/patrick/Library/Application Support/Copado/QConnect/execution/performance-testing/files/reporting/default.template' --html --xlsx --docx
Robot Framework Swarm: Reporter 
	Version 1.0.0 
self.args.ini:  /Users/patrick/Library/Application Support/Copado/QConnect/execution/performance-testing/files/configuration/RFSwarmReporter.ini 
	Configuration File:  /Users/patrick/Library/Application Support/Copado/QConnect/execution/performance-testing/files/configuration/RFSwarmReporter.ini 
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/Applications/QConnect.app/Contents/Resources/standalone_python/install/lib/python3.9/threading.py", line 973, in _bootstrap_inner
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/Applications/QConnect.app/Contents/Resources/standalone_python/install/lib/python3.9/threading.py", line 973, in _bootstrap_inner
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/Applications/QConnect.app/Contents/Resources/standalone_python/install/lib/python3.9/threading.py", line 973, in _bootstrap_inner
    self.run()
    self.run()
  File "/Applications/QConnect.app/Contents/Resources/standalone_python/install/lib/python3.9/threading.py", line 910, in run
    self.run()
  File "/Applications/QConnect.app/Contents/Resources/standalone_python/install/lib/python3.9/threading.py", line 910, in run
  File "/Applications/QConnect.app/Contents/Resources/standalone_python/install/lib/python3.9/threading.py", line 910, in run
    self._target(*self._args, **self._kwargs)
  File "/Applications/QConnect.app/Contents/Resources/standalone_python/install/lib/python3.9/site-packages/rfswarm_reporter/rfswarm_reporter", line 2156, in export_word
    self._target(*self._args, **self._kwargs)
  File "/Applications/QConnect.app/Contents/Resources/standalone_python/install/lib/python3.9/site-packages/rfswarm_reporter/rfswarm_reporter", line 2082, in export_xhtml
    self._target(*self._args, **self._kwargs)
  File "/Applications/QConnect.app/Contents/Resources/standalone_python/install/lib/python3.9/site-packages/rfswarm_reporter/rfswarm_reporter", line 2196, in export_excel
    self.display_message("Generating XHTML Report")
  File "/Applications/QConnect.app/Contents/Resources/standalone_python/install/lib/python3.9/site-packages/rfswarm_reporter/rfswarm_reporter", line 2072, in display_message
    self.display_message("Generating Word Report")
  File "/Applications/QConnect.app/Contents/Resources/standalone_python/install/lib/python3.9/site-packages/rfswarm_reporter/rfswarm_reporter", line 2072, in display_message
    self.display_message("Generating Excel Report")
  File "/Applications/QConnect.app/Contents/Resources/standalone_python/install/lib/python3.9/site-packages/rfswarm_reporter/rfswarm_reporter", line 2072, in display_message
    base.gui.updateStatus(msgout)
AttributeError: 'NoneType' object has no attribute 'updateStatus'
    base.gui.updateStatus(msgout)
AttributeError: 'NoneType' object has no attribute 'updateStatus'
    base.gui.updateStatus(msgout)
AttributeError: 'NoneType' object has no attribute 'updateStatus'

```

**Desktop (please complete the following information):**
 - OS: Mac OS

**Comment by damies13** [comment]
I Managed to reproduce this issue in Ubuntu 22.04, I expect it will behave the same on Windows - marking as a confirmed bug.

Thanks for the bug report :+1: 

As I mentioned in Discord the workaround for this is to use either `-n` or `--nogui` then the reports will be generated on the command line.

Assigning to next bugfix release

**Comment by damies13** [comment]
Fix merged into Release Branch v1.0.2, will be released soon
